src/refactorings/rename-symbol/rename-symbol.ts

```diff
@@ -13,7 +13,7 @@ async function renameSymbol(editor: Editor) {
   }
 }
 
-function findSelectedSymbol(editor: Editor): Symbol {
+function findSelectedSymbol(editor: Editor): Nothing {
```

Comment:
That doesn't seem right. `result` may be an `Identifier`. Both are implementing the `Symbol` interface, so the previous code should have been correct. What was the linter issue?